lib/readline.js

```diff
@@ -501,7 +501,7 @@ Interface.prototype._tabComplete = function(lastKeypressWasTab) {
 
       // If there is a common prefix to all matches, then apply that portion.
       var f = completions.filter(function completionFilter(e) {
```

Comment:
this could be reduced to `completions.filter(e => e)`.

Reply:
:+1: 

I was being cautiously. MDN indicates that the callback should return `true` or `false` and makes no mention of truthy or falsy values. But now I've looked at the ECMA-262 spec and it makes it explicit that the callback may return a value that is _coercible_ to a Boolean. I'll change it to your suggestion.

Reply:
(Might not bother with the change to arrow function just to keep the diff as small as possible. I assume there's no strong preference there.)

Reply:
I’d go for it. +1 -3 vs. +1 -1 are both really small, and I don’t think this line is interesting enough for people to use git blame on it either way.